llvm/tools/opt/CMakeLists.txt

```diff
@@ -29,24 +29,6 @@ set(LLVM_LINK_COMPONENTS
   Passes
   )
 
-# We don't want to link this into libLLVM
-add_llvm_library(LLVMOptDriver
-  STATIC
-  NewPMDriver.cpp
-  optdriver.cpp
-  PARTIAL_SOURCES_INTENDED
-  DEPENDS
-  intrinsics_gen
-)
-
-add_llvm_tool(opt
-  PARTIAL_SOURCES_INTENDED
-  opt.cpp
-  DEPENDS
-  intrinsics_gen
-  SUPPORT_PLUGINS
-
-  )
-target_link_libraries(opt PRIVATE LLVMOptDriver)
-
-export_executable_symbols_for_plugins(opt)
+add_subdirectory(lib)
```

Comment:
(can't comment above due to github limitations) should `LLVM_LINK_COMPONENTS` go down to `llvm/tools/opt/lib/CMakeLists.txt`?